checks/windows.py

```diff
@@ -330,3 +339,12 @@ def check32bitOn64bit(lines):
         # thx to secretply for the bugfix
         return [LEVEL_WARNING, "32-bit OBS on 64-bit Windows",
                 "You are running the 32 bit version of OBS on a 64 bit system. This will reduce performance and greatly increase the risk of crashes due to memory limitations. You should only use the 32 bit version if you have a capture device that lacks 64 bit drivers. Please run OBS using the 64-bit shortcut."]
+
+
+def checkEmulationStatus(lines):
+    if (len(search('Windows ARM64: Running with X64 Emulation', lines)) > 0):
+        return [LEVEL_WARNING, "X64 OBS on Windows ARM64",
+                "You are running the X64 version of OBS on an ARM64 Windows system. This will greatly reduce performance due to the mandatory use of emulation."]
+    elif (len(search('Rosetta translation used: true', lines)) > 0):
+        return [LEVEL_WARNING, "Intel OBS on Apple Silicon Mac",
+                "You are running the Intel version of OBS on an Apple Silicon Mac. This will greatly reduce performance due to the mandatory use of Rosetta."]
```

Comment:
Don't really agree with the wording. For some people it's virtually no difference, for some people it's massive. I'd go for something along the lines of "You may get improved performance using the Apple Silicon version of OBS"

Reply:
Wait, for Apple we can use softer wording. But for ARM64 Windows devices, the performance penalty is real.

Reply:
It's fine to have different wording for each platform.